docs/federate.md

```diff
@@ -0,0 +1,156 @@
+Setting up Federation
+=====================
+
+Federation is the process by which users on different servers can participate
+in the same room. For this to work, those other servers must be able to contact
+yours to send messages.
+
+The ``server_name`` configured in the Synapse configuration file (often
+``homeserver.yaml``) defines how resources (users, rooms,...) will be
+identified (ie: ``@user:example.com``, ``#room:example.com``). By
+default, it is also the default domain that other servers will use to
+try to reach your server via port 8448. This is easy to set
+up and will work provided you set the ``server_name`` to match your
+machine's public DNS hostname, and provide Synapse with a TLS certificate
+which is valid for your ``server_name``.
+
+For a more flexible configuration, you can have ``server_name``
```

Comment:
should all of this be under a "Delegation" section?

Reply:
Yes